index.js

```diff
@@ -7,16 +7,31 @@ async function run() {
     const projectUrl = core.getInput("project-url");
     const columnName = core.getInput("column-name");
     const labelName = core.getInput("label-name");
+    const milestoneName = core.getInput("milestone-name");
     const ignoreList = core.getInput("columns-to-ignore");
     const octokit = new github.GitHub(myToken);
     const context = github.context;
 
+    if(!milestoneName && !labelName){
```

Comment:
I think this is a good compromise/solution to allowing both labels and milestones to work with the action and not having them both be required. 👍